extra-libs/libquvi-scripts/autobuild/defines

```diff
@@ -3,3 +3,4 @@ PKGSEC=libs
 PKGDEP="lua-socket luabitop lua-expat"
 PKGDES="Library for parsing video download links"
 AUTOTOOLS_AFTER="--with-nsfw --with-geoblocked"
+ABSPLITDBG=0
```

Comment:
This package seems to only contain lua scripts.

- It should be marked noarch instead of turning off symbol split. 
- Since there is no binary linkage from anything in this package to external binaries, I'd say it's probably safe to skip rebuild for this package.